src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py

```diff
@@ -809,6 +829,7 @@ def __call__(
         )
 
         self._guidance_scale = guidance_scale
+        self._skip_layer_guidance_scale = skip_layer_guidance_scale
```

Comment:
need to add a decorator for this too https://github.com/huggingface/diffusers/blob/07d0fbf3ec255e3077e9e628bd740b50037f8c53/src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py#L642